### DISCUSSION TITLE
using deb repository seems to have an error in cofiguration

### DISCUSSION THREAD

**Discussion by auadams** [roxy-wi/roxy-wi#293] [General]
Hello,

Not sure if it was clear in the instructions after the "apt install" to do further configuration but the following was given after apt install. 

www-data       ALL=(ALL)       NOPASSWD: ALL
chown: cannot access 'roxy-wi.db': No such file or directory
dpkg: error processing package roxy-wi (--configure):
 installed roxy-wi package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 roxy-wi

for the time being i guess i will follow the site to find the missing step in the deb install so i can continue.

**Comment by Aidaho12** [comment]
Hello,
What version of Ubuntu do you use?  Is there more information, what exactly was failed... воскресенье, 03 октября 2021г., 11:31 +03:00 от Austin A  ***@***.*** :

**Reply by auadams** [reply]
Ubuntu 18.04.6 LTS (GNU/Linux 4.15.0-159-generic x86_64)

**Reply by auadams** [reply]
i managed to create the db invoking the ./create_db script after installing requirements.txt i'm going to try to do a fresh install and see it if happens again

**Comment by Aidaho12** [comment]
Try to use the latest version

**Comment by auadams** [comment]
Ubuntu(Bionic, Focal, Groovy, Hirsute) is listed as supported on your install website. But i will try with Focal

**Comment by auadams** [comment]
looks like the apt install went fine but https://10.1.1.21/app/overview.py is returning a blank page.

**Reply by auadams** [reply]
okay i fixed it. aparently the sudo apt install doesn't install all python requirements. you might want to either fix that or make a note about that within the install docs

**Comment by Aidaho12** [comment]
Yes, would be great, could you please describe your actions?

**Comment by auadams** [comment]
I installed a new instance of Focal (Not Bionic)

run apt install <packages>

after that i went into /var/www/haproxy-wi/app and ran pip3 install -r requirements.txt

**Comment by Aidaho12** [comment]
I see, thank you
